website/docs/plugin/framework/migrating/testing.mdx

```diff
@@ -0,0 +1,170 @@
+---
+page_title: 'Testing Migration: Migrating from SDKv2 to the Framework'
+description: >-
+  Write tests that verify that switching from SDKv2 to the Framework does not affect provider behavior.
+---
+
+# Testing
+
+During migration, you should [write tests](#testing-migration) to verify that the behaviour of your provider has not
+been altered by the migration itself. You will also need to [update](#provider-factories) your tests too.
+
+## Testing Migration
+
+During migration, we recommend writing tests to verify that switching from SDKv2 to the Framework has not affected your
+provider's behavior.
```

Comment:
```suggestion
During migration, we recommend writing tests to verify that switching from SDKv2 to the Framework has not affected your provider's behavior. These tests will generate a plan with the SDKv2 version of the provider, generate a corresponding plan with the Framework version of the provider, then compare the two plans to ensure that they are the same.
```

I think this section needed a bit more explanation to orient the reader.

Reply:
Totally agree that this needs a little more explanation. Have reworded, but I'm not sold on this so feel free to amend.

> During migration, we recommend writing tests to verify that switching from SDKv2 to the Framework has not affected your provider's behavior. These tests use identical configuration. The first test step applies a plan and generates state with the SDKv2 version of the provider. The second test step generates a plan with the Framework version of the provider and verifies that the plan is a no-op indicating that migrating to the framework has not altered behaviour.